File: src/main/java/br/com/leonardoferreira/jirareport/domain/vo/DynamicFieldConfig.java
package br.com.leonardoferreira.jirareport.domain.vo;

import lombok.Data;

import java.io.Serializable;

@Data
public class DynamicFieldConfig implements Serializable {
    private static final long serialVersionUID = -8677745120309321988L;

    private String name;

    private String field;

}

File: src/main/kotlin/br/com/jiratorio/domain/entity/DynamicFieldConfig.kt
package br.com.jiratorio.domain.entity

import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType
import javax.persistence.Id
import javax.persistence.JoinColumn
import javax.persistence.ManyToOne

@Entity
data class DynamicFieldConfig(

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long = 0,

    @ManyToOne
    @JoinColumn(name = "board_id", nullable = false)
    var board: Board,

    @Column(nullable = false)
    var name: String,

    @Column(nullable = false)
    var field: String

) : BaseEntity()
